ExNavigator.js

```diff
@@ -70,10 +66,11 @@ export default class ExNavigator extends React.Component {
     // mounted because it emits a didfocus event when it is mounted, before we
     // can get a ref to it
     if (!this._subscribedToFocusEvents) {
-      this._subscribeToFocusEvents(this);
+      this._subscribeToFocusEvents(navigator);
     }
 
     let scene = this._routeRenderer.renderScene(route, this);
+    scene = this.augmentScene(scene, route);
```

Comment:
Now I'm thinking this actually could be a prop (`this.props.augmentScene`). Then the owner of the ExNavigator (instead of its subclass) could write:
```js
render() {
  return <ExNavigator augmentScene={this._augmentScene} />;
}
@autobind
_augmentScene(scene, route) {
 ...
}
```